src/backend/api/Fusion.Resources.Logic/Requests/UnauthorizedWorkflowException.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace Fusion.Resources.Logic.Requests
+{
+    [Serializable]
+    public class UnauthorizedWorkflowException : Exception
```

Comment:
Any point including all these constructors when they are never used? Is this som resharper magic template?

Reply:
It's from the default VS functionality when generating a class which ends with Exception. I guess the template is based on this advise: https://docs.microsoft.com/en-us/dotnet/standard/exceptions/best-practices-for-exceptions#include-three-constructors-in-custom-exception-classes